cache/src/main/java/com/uber/buckcache/datastore/impl/ignite/LocalCacheEventListener.java

```diff
@@ -27,6 +26,13 @@ public LocalCacheEventListener(IgniteCache<Long, byte[]> metadataCache, IgniteCa
 
   @Override
   public boolean apply(CacheEvent evt) {
+    String cacheEventName = EVENT_TYPE_TO_NAME_MAP.get(evt.type());
+    if (cacheEventName != null && cacheEventName.length() > 0) {
+      String cacheEventMetric = String.format("event_cache_%s_%s_count", evt.cacheName(), cacheEventName).toLowerCase();
```

Comment:
@anoopc1 nit: do you think `event_cache.%s.%s.count` looks better? matches how we namespace datadog metrics elsewhere